File: Csharp/Alphabet symmetry.cs
// https://www.codewars.com/kata/59d9ff9f7905dfeed50000b0/train/csharp
using System.Linq;
using System.Collections.Generic;

public static class Kata
{
  public static List<int> Solve(List<string> arr)
  {
    return arr.Select(s => s.Where((c, i) => c % 32 == i + 1).Count()).ToList();
  }
}

File: javascript/Alphabet symmetry.js
// https://www.codewars.com/kata/59d9ff9f7905dfeed50000b0/train/javascript
function solve(arr){  
    const abc = "abcdefghijklmnopqrstuvwxyz";
    return arr.map(e=>e.toLowerCase().split('').filter((a,b)=> b == abc.indexOf(a)).length)
  };